prototype2/multigrid/mgmesytec/DataParserTest.cpp

```diff
@@ -10,7 +10,11 @@ static Producer producer {"noserver", "nostream"};
 class MesytecDataTest : public TestBase {
 protected:
   NMXHists hists;
-  MesytecData mesytec{0, "nofile", 1}; // Dont dumptofile select module with 20 depth in z
```

Comment:
This is to allow tests to build without `-DDUMPTOFILE` enabled. It tests on Jenkins with the flag enabled, so this may go unnoticed if you don't also build tests locally with different configs.